accounts/static/accounts/css/login.css

```diff
@@ -215,17 +213,30 @@ body {
 
 /* ADAPTIVE  for mobile devices */
 
-@media only screen and (min-width: 320px) and (max-width: 600px) {
+@media only screen and (width >= 320px) and (width <= 600px) {
+  a {
+    font-size: 5px;
+  }
+
+  h1 {
+    font-size: 20px;
+  }
+
+  .span {
+    font-size: 3px;
+  }
+
   .container {
     background-color: #fff;
     border-radius: 30px;
-    box-shadow: 0 5px 15px rgba(0, 0, 0, 0.35);
+    box-shadow: 0 5px 15px rgb(0 0 0 / 35%);
```

Comment:
![image](https://github.com/user-attachments/assets/3668a648-01c8-4c17-8494-b73a7e21d893)
gpt said yes which OBVIOUSLY means it is Cool